client/js/components/drawer.js

```diff
@@ -0,0 +1,25 @@
+class Drawer {
+  constructor(el) {
+    this.el = el;
+    this.trigger = el.getElementsByClassName('js-trigger')[0];
+    this.trigger.addEventListener('click', this.toggleActive.bind(this));
```

Comment:
Yea - IE8 being one that won't, and I would heavily suggest for our sanity that it gets nowhere near the mustard.

(but it still get's the core content).